contracts/yield-source/YearnV2YieldSource.sol

```diff
@@ -248,7 +300,7 @@ contract YearnV2YieldSource is IYieldSource, ERC20Upgradeable, OwnableUpgradeabl
     /// @param tokens amount of tokens to be converted
     /// @return shares number of shares equivalent to the amount of tokens
     function _tokenToShares(uint256 tokens) internal view returns (uint256 shares) {
-        if(totalSupply() == 0) {
+        if (totalSupply() == 0) {
```

Comment:
let's pull this into a variable

Reply:
Done: https://github.com/pooltogether/pooltogether-yearnv2-yield-source/pull/9/commits/32b130dbbaf28b614075ce2d30b0ee7c9f47593c